.tool-versions

```diff
@@ -0,0 +1 @@
+python 3.8.18 3.9.18 3.10.13 3.11.6 pypy3.9-7.3.12
```

Comment:
same question as in python-version, is there a need to mention the micro version? couldn' t it be just 3.X instead of 3.X.Y?

Reply:
Answered at https://github.com/bndr/pipreqs/pull/409#discussion_r1383328090.